src/erfadatextra.c

```diff
@@ -0,0 +1,38 @@
+#include "erfa.h"
+#include "erfaextra.h"
+
+static eraLEAPSECOND *changes;
+static int NDAT = 0;
```

Comment:
Maybe this should be initialized with `-1` (and tested for -1 in `eraDatInit`)? It may be an exotic case, but someone may want to switch off leap seconds at all by providing an empty list. I see no formal reason why this should be excluded.

Reply:
I agree it seems logical that one should be able to pass in `NDAT=0`, but a quick inspection of `dat.c` shows that for that case all calls with return error code `-5`.... Of course, even with just one no-op entry, you'd get warnings about dates being too far in the future, etc.

Overall, my sense is that it is best to keep this handle as simple as possible; that said, the least I can do is to use any `NDAT <= 0` to initialize the table.